implementations/c/include/ockam/transport.h

```diff
@@ -1,147 +1,168 @@
 /**
  ********************************************************************************************************
  * @file        transport.h
- * @brief       Public-facing API function prototypes for Ockam's transport library
+ * @brief       Public-facing API function prototypes for Ockam's transport
+ *library
  ********************************************************************************************************
  */
 #ifndef OCKAM_TRANSPORT_H
 #define OCKAM_TRANSPORT_H 1
 
 /*
  ********************************************************************************************************
- *                                             INCLUDE FILES                                            *
+ *                                             INCLUDE FILES *
  ********************************************************************************************************
  */
+#include <arpa/inet.h>
+#include <errno.h>
+#include <netdb.h>
+#include <netinet/in.h>
+#include <netinet/tcp.h>
+#include <stdint.h>
 #include <stdlib.h>
 #include <string.h>
-#include <stdint.h>
+#include <sys/socket.h>
+#include <sys/types.h>
```

Comment:
most of these can likely move to an implementations, except `stdint.h`

Reply:
done

Reply:
In `transport.h` there is:
```
#include <stdlib.h>
#include <string.h>
```
Do we need them?